src/components/WindowEditor.tsx

```diff
@@ -23,21 +23,24 @@ interface EditorProps {
     language?: 'tsx' | 'jsx' | 'md' | 'mdx'
     className?: string
     style?: React.CSSProperties
+    withHeader?: boolean
```

Comment:
I think we could customize the header name, then turn it into an object with this properties

```suggestion
    header: { show: boolean; title: string }
```